cluster/manifests/role-sync-controller/cronjob.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if .Cluster.ConfigItems.role_sync_controller_enabled "true" }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: role-sync-controller
+  labels:
+    application: kubernetes
+    component: role-sync-controller
```

Comment:
should be in `kube-system`

Needs also RBAC role to read namespaces/create roles

Reply:
I was wondering about using `kube-system` 😅 
I'll add the RBAC as well. Thanks 🙏

Reply:
Added in the next commit.